crates/binjs_io/src/entropy/dictionary.rs

```diff
@@ -291,6 +291,24 @@ where
     }
 }
 
+/// Options for creating a dictionary.
+#[derive(Clone, Debug, Serialize, Deserialize, Default)]
+pub struct Options {
+    depth: usize,
+    width: usize,
+}
+impl Options {
+    /// Set the length of AST paths used to establish probabilities.
+    pub fn with_depth(self, depth: usize) -> Self {
+        Options { depth, ..self }
+    }
+
+    /// Set the width of windows of references used to establish probabilities.
```

Comment:
It would be better moving the comment about the meaning of `depth`/`width` to fields above.